requirements/requirements.server.txt

```diff
@@ -57,7 +57,7 @@ cryptography==44.0.2
     #   pyjwt
     #   pyspnego
     #   requests-kerberos
-cx-oracle==8.3.0
+oracledb==3.1.1
```

Comment:
This would replace the client used by sql alchemy in the production to communicate with oracle. Need to check if there are special hints provided to sqla for oracle and whether they will still be supported by the new client.